app/models/chef.rb

```diff
@@ -1,4 +1,9 @@
 class Chef < ApplicationRecord
-   validates_presence_of :name
-   has_many :dishes
-end
\ No newline at end of file
+  validates_presence_of :name
+  has_many :dishes
+  has_many :ingredients, through: :dishes
+
+  def unique_ingredients
+    ingredients.distinct
```

Comment:
Love it! Great use of the additional association to make this ingredients list much more easily